airbyte-integrations/connector-templates/source-python/README.md.hbs

```diff
@@ -57,7 +57,7 @@ docker run --rm -v $(pwd)/airbyte-integrations/connectors/source-{{dashCase name
 
 ### Integration Tests
 1. From the airbyte project root, run `./gradlew :airbyte-integrations:connectors:source-{{dashCase name}}:standardSourceTestPython` to run the standard integration test suite.
-1. To run additional integration tests, place your integration tests in the `integration_tests` directory and run them with `pytest integration_tests`.
+1. To run additional integration tests, place your integration tests in a new directory `integration_tests` and run them with `pytest integration_tests`.
```

Comment:
Should we have a gradle rule instead? (and explain that it runs pytest behind the scene?

Does it mean that the tests don't run in docker?

Reply:
my understanding of how we are handling non-standard test integration tests is poor. i think we need to document a bit more what that environment looks like. how do you add deps to it (is it same deps as test)? etc. fine with it not being in this PR as it's a bit of a scope creep. but if it is indeed under-documented can you create an issue to document what we need to here.

Reply:
I like the idea of a gradle rule, such that if they implement things in a particular it works automatically with `integrationTest` task

Reply:
created issue: https://github.com/airbytehq/airbyte/issues/1371